server/modules/surveyRdb/manager/surveyRdbManager.js

```diff
@@ -100,22 +102,35 @@ export const queryTable = async (
       rows.map(async (row) => {
         const recordUuid = row[DataTable.colNameRecordUuuid]
         const record = await RecordRepository.fetchRecordByUuid(surveyId, recordUuid)
-        const parentNodeUuid = R.prop(`${NodeDef.getName(nodeDefTable)}_uuid`, row)
+        const parentNodeUuid = R.prop(RdbDataView.getColUuid(nodeDefTable), row)
```

Comment:
RdbDataView.getColUuid is deprecated. use ViewDataNodeDef

Reply:
ok